plugins/org/src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import {
+  Avatar as MaterialAvatar,
+  createStyles,
+  makeStyles,
+  Theme,
+} from '@material-ui/core';
+import { CSSProperties } from '@material-ui/core/styles/withStyles';
```

Comment:
just `from '@material-ui/styles'` is the main export i think

Reply:
Afaik it should be grabbed from `react` if possible